install_config/aggregate_logging.adoc

```diff
@@ -1518,3 +1518,602 @@ $ oc label node --all logging-infra-fluentd=true <1>
 ----
 <1> This example assumes use of the default Fluentd node selector and it being
 deployed on all nodes.
+
+ifdef::openshift-origin[]
+[[exported-fields]]
+== Exported Fields
+
+These are the fields exported by the logging system and available for searching
+from Elasticsearch and Kibana. Use the full, dotted field name when searching.
+For example, for an Elasticsearch */_search URL*, to look for a Kubernetes pod name,
+use `/_search/q=kubernetes.pod_name:name-of-my-pod`.
+The following sections describe fields that may not be present in your logging store.
+Not all of these fields are present in every record.
+The fields are grouped in the following categories:
+
+* `exported-fields-Default`
+* `exported-fields-rsyslog`
+* `exported-fields-systemd`
+* `exported-fields-kubernetes`
+* `exported-fields-docker`
+* `exported-fields-pipeline_metadata`
+* `exported-fields-ovirt`
+* `exported-fields-aushape`
+* `exported-fields-tlog`
+
+[discrete]
+[[exported-fields-Default]]
+=== Top Level Fields
+
+The top level fields are common to every application, and may be present in every record.
+For the Elasticsearch template, this is what populates the actual mappings of `default` in
+the template's mapping section.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
```

Comment:
I also find @timestamp.raw, it seems it is newly added, shall we need to add it?